docs/output.md

```diff
@@ -342,6 +342,120 @@ The saved files are the _good_ files, passing the `dust` or `entropy` filter tre
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `metagenomics_screening/profiling/malt/`
+  - `<sample_id>.rma6`: binary file containing all alignments and taxonomic information of hits that can be loaded into the [MEGAN6](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/megan6/) interactive viewer
+  - `<sample_id>.blastn.sam`: sparse SAM file containing alignments of each hit (if `--metagenomics_malt_savereads`)
+  - `*.log`: LOG file containing runtime log of MALT. NOTE: If you are running parallel malt runs with `--metagenomics_malt_group_size` set above 0, your log files will be labelled with the name of _one_ of the input files run for each of the parallel executions.
+
+</details>
+
+MALT is a metagenomic aligner (equivalent to BLAST, but much faster). It produces direct alignments of sequencing reads in a reference genome. It is often used for metagenomic profiling or pathogen screening, and specifically in nf-core/eager, of off-target reads from genome mapping.
+
+You will receive output for each library. This means that if you use TSV input and have one library sequenced over multiple lanes and sequencing types, these are merged and you will get mapping statistics of all lanes and sequencing configurations in one value.
+
+The main output of MALT is the `.rma6` file format, which can be only loaded into MEGAN and it's related tools.
+
+You will only receive the `.sam` files if you supply `--metagenomics_malt_savereads` parameters to the pipeline.
+
+### MetaPhlAn
+
+[MetaPhlAn](https://github.com/biobakery/metaphlan) is a computational tool for profiling the composition of microbial communities (Bacteria, Archaea and Eukaryotes) from metagenomic shotgun sequencing data (i.e. not 16S) with species-level resolution via marker genes.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `metagenomics_screening/profiling/metaphlan/`
+  - `<sample_id>.biom`: taxonomic profile in BIOM format
+  - `<sample_id>.bowtie2out.txt`: BowTie2 alignment information (can be re-used for skipping alignment when re-running MetaPhlAn with different parameters)
+  - `<sample_id>_profile.txt`: MetaPhlAn taxonomic profile including abundance estimates
+
+</details>
+
+The main taxonomic profiling file from MetaPhlAn is the `*_profile.txt` file. This provides the abundance estimates from MetaPhlAn however does not include raw counts by default. Intermediate Bowtie2 output `.bowtie2out.txt`, which presents a condensed representation of the mapping results of your sequencing reads to MetaPhlAn's marker gene sequences. The alignments are listed in tab-separated columns, including Read ID and Marker Gene ID, with each alignment represented on a separate line.
+
```

Comment:
As you mention taxpasta in Kraken2 section, maybe you want to to mention this here too?

Reply:
done